scripts/lib/upgrade-zulip-stage-2

```diff
@@ -82,6 +87,25 @@ try:
 except (configparser.NoSectionError, configparser.NoOptionError):
     tornado_processes = 1
 
+# postgresql.version is effectively required, but wasn't previously;
+# fill it in based on what the OS provides.
+try:
+    postgres_version = int(config_file.get('postgresql', 'version'))
+except (configparser.NoSectionError, configparser.NoOptionError):
+    default_postgres_version = {
+        ('debian', '10'): 11,
+        ('ubuntu', '18.04'): 10,
+        ('ubuntu', '20.04'): 12,
+        ('centos', '7'): 11,
```

Comment:
These probably want to be strings rather than integers, so we don’t accidentally install Postgres 12.399999999999999 some day.

Reply:
hahahah yeah good call :)